### PR TITLE
T-Tabular: New feature and bug fixes related to xls-like files.

### PR DESCRIPTION
**Pull request by skodape**
New feature and bug fixes related to xls-like files.

2.2.1 - Fixed bug with missing trailing columns in xls-like files with empty leading columns and header autogeneration.
2.2.0 - Added possibility to strip null value in header. For XSL: If row is shorter than header then row is expanded.

### CODE REVIEW

File: t-tabular/pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	<artifactId>uv-t-tabular</artifactId>
 	<name>T-Tabular</name>
 	<description>Convert tabular data into rdf data.</description>
-	<version>2.1.1-SNAPSHOT</version>
+	<version>2.2.1-SNAPSHOT</version>
```
Comment (skrchnavy) [inline t-tabular/pom.xml]: Not sure why we have to skip version 2.2.0. Is there special reason not to release it?

File: t-tabular/README.md [hunk 2]
```diff
@@ -35,6 +35,8 @@
 
 |Version          |Release notes               |
 |-----------------|----------------------------|
+|2.2.1 | Fixed bug with missing trailing columns in xls-like files with empty leading columns and header autogeneration. |
+|2.2.0 | Added possibility to strip null value in header. For XSL: If row is shorter than header then row is expanded. |
```
Comment (skrchnavy) [inline t-tabular/README.md]: what is difference in 2.2.0 and 2.1.0, from this description same change was included here

**Comment by tomas-knap** [comment]
@skrchnavy - Ad versions, Petr wanted to have two versions inside one release period. So now the version of this DPU and the CUNI version of tabular (created just because we forced Petr to adjust versions, which is a bit crazy) diverged a bit - this pull request should put it again together. So I suggest do not think about the versions too much  and simply use the latest version as the current latest DPU version. Period.

**Comment by skrchnavy** [comment]
Then we need version history to contain real release versions and not those that were not released. So it have to be properly documented what is included in which version.

**Pull request merged**